docs/community/team.md

```diff
@@ -0,0 +1,18 @@
+# GradleUp Team
+
+The project is led by a small team,
+with a consensus-based decision making and governance.
+There is no special governance charter at the moment.
```

Comment:
Nit: I would on the cool stuff more than what we don't have yet. Maybe something like so?

```suggestion
The project is led by a small team of passionate Gradle users and contributors.
The team is decentralized and decisions are made based on consensus and common sense.
```